docs/ref/models/lookups.txt

```diff
@@ -213,6 +213,13 @@ following methods:
         The name of this lookup, used to identify it on parsing query
         expressions. It cannot contain the string ``"__"``.
 
+    .. attribute:: prepare_rhs
+
+        Whether :attr:`rhs`, that is a plain value, should be prepared for use
+        as a parameter in a query. Calls ``lhs.output_field.get_prep_value()``
+        if defined, or wraps ``rhs`` in :class:`Value()
+        <django.db.models.Value>` otherwise. Defaults to ``True``.
```

Comment:
I don't know offhand what does `prepare_rhs` do, I know I can check the source code, but considering this the public facing docs, I would suggest to be a bit more verbose. In the following I'm assuming `prepare_rhs` is a bool :-)

```suggestion
        Indicates whether the  :attr:`rhs` attribute, which is a plain value,
        should be prepared for use as a parameter in a query. It calls
        ``lhs.output_field.get_prep_value()`` if set, or wraps `rhs`` in
        :class:`Value() <django.db.models.Value>` otherwise. The default value
        is ``True``.
```

Reply:
`lhs.output_field.get_prep_value()` is called when `output_field` is defined on `lhs` and `get_prep_value()` is defined on `lhs.output_field`, otherwise `rhs` is wrapped in `Value()`:

https://github.com/django/django/blob/ea596a52d9f905596cc5335930c8f2ac4511204c/django/db/models/lookups.py#L86-L90

Your proposition suggests that `lhs.output_field.get_prep_value()` is called when `prepare_rhs` is `True`, or `rhs` is wrapped with `Value()` when `prepare_rhs` is `False`.

I tried to improve the wording.